packages/plugin-hardhat/hardhat.config.js

```diff
@@ -1,20 +1,20 @@
 require('@openzeppelin/hardhat-upgrades');
 
+const settings = {
```

Comment:
Can you remove these settings, they shouldn't be necesary with the new `extendConfig` code.

Reply:
Those were there before, should I add them to the extendConfig?